pkg/configurator/client.go

```diff
@@ -55,6 +55,9 @@ const (
 
 	// enablePrivilegedInitContainer is the key name used to specify whether init containers should be privileged in the ConfigMap
 	enablePrivilegedInitContainer = "enable_privileged_init_container"
+
+	// enableTicker is a flag to configure ticker for regular proxy broadcast updates
+	enableTicker = "enable_ticker"
```

Comment:
Can we instead change this to an optional `config_resync_duration`, and disable ticker if this is 0s|m|h duration?

Reply:
yeah np

Reply:
`duration` to me implies how long the config resync should take (from the begin to the end).
I'd use `interval` or `period between resyncs`.

Reply:
`config_resync_interval` it is